tests/behat/behat-features/src/PushingContext.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+/**
+ * @file
+ * The behat context for replication tests.
+ */
+
+use Behat\Mink\Exception\ExpectationException;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+use GuzzleHttp\Client;
+
+/**
+ * Defines features from the pushing context.
+ */
+class PushingContext extends RawDrupalContext {
+
+  /**
+   * Random suffix to use in tests.
+   *
+   * @var string
+   */
+  protected $randomSuffix;
+
+  /**
+   * I open the contentpool.
+   *
+   * @Given I open the contentpool
+   */
+  public function openContentpool() {
+    $this->visitContentpoolPath('/');
+  }
+
+  /**
+   * I visit path on contentpool.
+   *
+   * @param string $path
+   *   Given path.
+   *
+   * @Given I visit path :path on contentpool
+   */
+  public function visitContentpoolPath($path) {
+    $this->setMinkParameter('base_url', 'http://contentpool-project.localdev.space');
```

Comment:
this seems wrong, base url is the satellite?  - it should read the contentpool url environment variable if any, I think we have one in wv-* projects, let's just use the same variable name.